src/pretix/base/views/metrics.py

```diff
@@ -1,27 +1,33 @@
 from django.conf import settings
-from http import HttpResponse
+from django.http import HttpResponse
 from .. import metrics
 
-unauthedResponse = HttpResponse("<html><title>Forbidden</title><body>You are not authorized to view this page.</body></html>", mimetype="text/html")
-unauthedResponse["WWW-Authenticate"] = 'Basic realm="metrics"'
-unauthedResponse.status_code = 401
+
+def unauthedResponse():
+    content = "<html><title>Forbidden</title><body>You are not authorized to view this page.</body></html>"
+    unauthedResponse = HttpResponse(content, mimetype="text/html")
+    unauthedResponse["WWW-Authenticate"] = 'Basic realm="metrics"'
+    unauthedResponse.status_code = 401
```

Comment:
Also, am I missing something regarding the camelCasing here? `get_unauthed_response` and `unauthed_response` would be more in line with the code base.